### PR TITLE
added info to resume

### PR DESCRIPTION
**Pull request by VysotskiVadim**

### CODE REVIEW

**Comment by VysotskiVadim** [comment]
TODO: 

- [ ] Check line endings, should be windows!

File: RESUME.txt [hunk 1]
```diff
@@ -1 +1,52 @@
...
+Resume of Vadzim Vysotski. Software engineer.
+This version could be outdated, the most resent could be found here: https://github.com/VysotskiVadim/resume
+
+Contacts:
+    Mobile, Telegram: +375 44 73 444 53
+    Skype: vadzim_vysotski
+    Email: vysotski.vadim@gmail.com
+
+Education:
+Bachelor's degree in Computer Science, BSUIR, 2011 - 2016
```
Comment (LiudmilaVys) [inline RESUME.txt]: Aren't you a specialist? Bachelor's degree is lower than specialists.

File: RESUME.txt [hunk 2]
```diff
@@ -1 +1,52 @@
...
+    Used intensively: C#, .NET
```
Comment (LiudmilaVys) [inline RESUME.txt]: Do not hesitate adding Haxe.

Comment (VysotskiVadim) [reply RESUME.txt]: I described here what I used when I was at university.

File: RESUME.txt [hunk 3]
```diff
@@ -1 +1,52 @@
...
+    Keurig(2016-2017): mobile application to control brewer
+    Used intensively: C#, Android, iOS, Xamarin.
+    Used less so: JS, Java
+    Done:
+        * Improved custom navigation framework from previous project, added more features like shared element transitions, screens hierarchy;
+        * Integrated third party Android/iOS library in common logic, reported libraries code issues which affects our features;
+        * Was a mentor for new developers, helped them quickly understand project structure and start working on features in short time;
+        * Integrated existing coffee web store to application via WebView, added JS API for web store team to call mobile native features.
+
+Developer, Playtika, 2017 - 2019
```
Comment (LiudmilaVys) [inline RESUME.txt]: It might be better to defined months also.

File: RESUME.txt [hunk 4]
```diff
@@ -1 +1,52 @@
...
+Projects:
+    PH: poker mobile game
+    Used intensively: Haxe, Monosyne (in-house game engine)
+    Used less so: C#, JS, Gradle, Android, iOS
+    Done:
+        * Implemented features which requires synchronization of few microservices, animations and user actions;
+        * Improved project stability by refactoring which fixed ANR and crashes;
+        * Did code review and was reviewed by teammates;
+        * Supported company infrastructure libraries: fixed C# specific issues in Haxe code, setup unit test to cover cs target;
+        * Improved CI CD to support web build and deployments.
```
Comment (LiudmilaVys) [inline RESUME.txt]: Do not forget to mention development of debugger for Haxe.

File: RESUME.txt [hunk 5]
```diff
@@ -1 +1,52 @@
...
+Developer, Epam Systems, 2014 - 2017
+Projects:
+    Plex(2014- 2015): Migrating old ERP system from VBS HTML JS to new technology stack: ASP.NET MVC 4 + XSL-FO (for PDF generation) + Knockout.
+    Used intensively: C#, XSLT, XSL-FO, T-SQL
+    Used less so: JS, Knockout
+    Done:
+        * Migrated a lot of documents, meet all deadlines;
+        * Implemented tool to compare old and new(migrated old) documents, for pixel perfect migration, which saved a lot of time for all team.
+    
+    CHA(2015): mobile application for airport's host assistance to have info about flights and passengers
```
Comment (LiudmilaVys) [inline RESUME.txt]: Let's disclose abbreviation here.

File: RESUME.txt [hunk 6]
```diff
@@ -1 +1,52 @@
-TODO: write resume. 
\ No newline at end of file
+Resume of Vadzim Vysotski. Software engineer.
+This version could be outdated, the most resent could be found here: https://github.com/VysotskiVadim/resume
```
Comment (LiudmilaVys) [inline RESUME.txt]: Please replace this link with another which leads directly to resume, not just to your repo.

File: RESUME.txt [hunk 7]
```diff
@@ -1 +1,52 @@
...
+
+Contacts:
+    Mobile, Telegram: +375 44 73 444 53
```
Comment (LiudmilaVys) [inline RESUME.txt]: Phone format is not very international, but it's up to you.

File: RESUME.txt [hunk 8]
```diff
@@ -1 +1,52 @@
...
+    Skype: vadzim_vysotski
```
Comment (LiudmilaVys) [inline RESUME.txt]: Is it possible to format this contact to make it clickable and open it directly in skype?

File: RESUME.txt [hunk 9]
```diff
@@ -1 +1,52 @@
...
+    Used less so: JS, Java
+    Done:
+        * Improved custom navigation framework from previous project, added more features like shared element transitions, screens hierarchy;
+        * Integrated third party Android/iOS library in common logic, reported libraries code issues which affects our features;
+        * Was a mentor for new developers, helped them quickly understand project structure and start working on features in short time;
+        * Integrated existing coffee web store to application via WebView, added JS API for web store team to call mobile native features.
+
+Developer, Playtika, 2017 - 2019
+Projects:
+    PH: poker mobile game
```
Comment (LiudmilaVys) [inline RESUME.txt]: Let's disclose abbreviation here.

File: RESUME.txt (hunk 1 above)
Comment (VysotskiVadim) [reply RESUME.txt]: fixed

File: RESUME.txt (hunk 5 above)
Comment (VysotskiVadim) [reply RESUME.txt]: done, thanks

File: RESUME.txt (hunk 6 above)
Comment (VysotskiVadim) [reply RESUME.txt]: fixed, thanks

File: RESUME.txt (hunk 8 above)
Comment (VysotskiVadim) [reply RESUME.txt]: no, it's just a plain text

File: RESUME.txt (hunk 9 above)
Comment (VysotskiVadim) [reply RESUME.txt]: done, thank you

File: RESUME.txt (hunk 3 above)
Comment (VysotskiVadim) [reply RESUME.txt]: no, I believe year is enough

**Pull request merged**